### ISSUE TITLE
OverrideFieldOrPropertyMulticastAspect fails to compile

### ISSUE THREAD

**Issue by jon-armen** [postsharp/Metalama.Extensions#40]
Attempted to create a simple aspect to override a property, however it fails with multicasting.  The same code without Multicasting works.

error LAMA0222: The compile-time project could not be compiled. In most cases, this is due to a problem in your code and can be diagnosed using the other reported errors. If, however, you believe this is due to a bug in Metalama, please report the issue and include diagnostic information available in ...

```
public sealed class MyAspectAttribute : OverrideFieldOrPropertyMulticastAspect
{
    public override dynamic? OverrideProperty { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
}
```

**Comment by prochan2** [comment]
Hello Jon, thank you for getting it touch. We're going to have a look at this, and we'll get back to you soon.

**Comment by gfraiteur** [comment]
Hello,

I'm sorry, it seems this ticket slipped through the cracks of our tracking system.
Is the issue still actual?
Could you attach the exception file mentioned in the error message?

**Comment by svick** [comment]
This was fixed in Metalama 2023.3.5-rc.
